lib-api/src/main/java/org/ergoplatform/appkit/babelfee/BabelFeeBoxBuilder.java

```diff
@@ -0,0 +1,60 @@
+package org.ergoplatform.appkit.babelfee;
+
+import org.ergoplatform.appkit.Address;
+import org.ergoplatform.appkit.ErgoId;
+import org.ergoplatform.appkit.SigmaProp;
+
+import java.util.Objects;
+
+public class BabelFeeBoxBuilder {
```

Comment:
Please provide JavaDoc for the class and all the methods.

Reply:
done